app/views/shared/_registration_steps.html.erb

```diff
@@ -0,0 +1,12 @@
+<% steps = { form: 1, confirm_email: 2, waiting_for_moderator: 3, good_to_go: 4 } %>
+<% step ||= :form %>
+<% step_number = steps[step] %>
+<div class="registration-steps mb-1" style="font-size: 9px">
+  <%= steps.map do |step, number|
+    css = ""
```

Comment:
Not a big deal, but why didn't you use the helper?

Reply:
 yeah I agree but this module will 100% be re-written when we get UI design, so let's keep as is